src/utils/supabase/middleware.test.ts

```
@@ -0,0 +1,592 @@
+          origin: 'http://localhost:3000'
+        } as any
+      }
+
+      const response = await middleware(mockRequest as NextRequest)
+      expect(response).toBeDefined()
+    })
+
+    it('should handle missing environment variables gracefully', async () => {
+      const originalEnv = process.env
+      process.env = { ...originalEnv }
+      delete process.env.NEXT_PUBLIC_SUPABASE_URL
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)